src/alert.scss

```diff
@@ -10,41 +10,64 @@
 $block: #{$fd-namespace}-alert;
 
 .#{$block} {
-  // SETTINGS
-  $fd-alert-color: fd-color("text", 1) !default;
-  $fd-alert-border-color: fd-color("neutral", 4) !default;
-  $fd-alert-border-color--warning: fd-color("status", 2) !default;
-  $fd-alert-border-color--error: fd-color("status", 3) !default;
-  $fd-alert-border-color--success: fd-color("status", 1) !default;
-  $fd-alert-border-color--information: fd-color("action", 1) !default;
-  $fd-alert-action-color: fd-color("action", 1) !default;
-
-  $fd-alert-background-color: fd-color-state("default") !default;
-  $fd-alert-background-color--warning: fd-color-state("alert") !default;
-  $fd-alert-background-color--error: fd-color-state("negative") !default;
-  $fd-alert-background-color--success: fd-color-state("positive") !default;
-  $fd-alert-background-color--information: fd-color-state("information") !default;
-
-  --fd-alert-background-color: var(--fd-color-background-default);
-  --fd-alert-border-color: var(--fd-color-neutral-4);
-  --fd-alert-action-color: var(--fd-color-action-1);
-
-  $fd-alert-padding-x: fd-space(2.25) !default;
-  $fd-alert-padding-y: fd-space("xs") !default;
+  $fd-alert-border: 1px solid;
+  $fd-alert-border-radius: 0.25rem;
+  $fd-alert-padding: 0.5rem 1rem;
+  $fd-alert-padding-default: 1rem;
+  $fd-alert-padding-dismissible: 2.5rem;
+  $fd-alert-padding-status: 2.5rem;
+  $fd-alert-text-color: var(--sapTextColor, $sapTextColor);
+
+  @mixin fd-alert-icon-container {
+    position: absolute;
+    width: 2.5rem;
+    top: 0.625rem;
+    left: 0;
+  }
+
+  @mixin fd-alert-close-btn-container {
+    position: absolute;
+    width: 2rem;
+    height: 1.625rem;
+    top: 0.375rem;
+    right: 0.125rem;
+  }
+
+  // SUCCESS
+  $fd-alert-background-color--success: var(--sapSuccessBackground, $sapSuccessBackground) !default;
```

Comment:
These can just be `var(--sapSuccessBackground)`, postcss will do the rest

Reply:
Removed the sass variables